src/components/dateTimePicker/index.js

```diff
@@ -61,6 +61,22 @@ class DateTimePicker extends BaseComponent {
      * The time format for the text display
      */
     timeFormat: PropTypes.string,
+    /**
+     * Allows changing of the locale of the component (iOS only)
+     */
+    locale: PropTypes.string,
+    /**
+     * Allows changing of the time picker to a 24 hour format (Android only)
+     */
+    is24Hour: PropTypes.bool,
+    /**
+     * The interval at which minutes can be selected. Possible values are: 1, 2, 3, 4, 5, 6, 10, 12, 15, 20, 30 (iOS only)
+     */
+    minuteInterval: PropTypes.number,
+    /**
+     * Allows changing of the timeZone of the date picker. By default it uses the device's time zone (iOS only)
+     */
+    timeZoneOffsetInMinutes: PropTypes.number,
```

Comment:
spread the whole `RNDateTimePicker` prop types 
`...RNDateTimePicker.propTypes` should do the trick